setup.py

```diff
@@ -39,14 +39,14 @@
     long_description=long_description,
     keywords='dict, dictionary, mapping, bidirectional, bijection, bijective, injective, two-way, 2-way, double, inverse, reverse',
     url='https://github.com/jab/bidict',
-    license='ISCL',
+    license='ISC',
```

Comment:
Looking through the first page of results for https://github.com/search?utf8=%E2%9C%93&q=filename%3Asetup.py+license+iscl&type=Code&ref=searchresults I see plenty of `license='ISCL'` as well as `license='ISC'` and`license='ISC License (ISCL)'`. Can you provide a reference for why your proposed change is right and the others are wrong? Another quick search I just did for "setup.py license values" turned up http://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/creation.html which actually advises that `license` should be set to the filename containing the license, which is news to me.